blog/zk-snark-pallet.md

```diff
@@ -0,0 +1,263 @@
+# Zk-SNARKs with Substrate (Part 3)
+
+This is the last part of a series of articles about zk-SNARKs. If you haven't read the previous ones (Part 1, Part 2), I encourage you to do so now.
+
+In our first article, we defined Bob's problem. He creates a contest, where the first person who solves the equation: 
+
+<center>
+
+$$ x^2+3=12 $$
+
+</center>
+
+will be able to join his Bright Coders union. As you probably remember, Alice was one of his friends who knows the solution. She was only afraid that revealing it loudly, could encourage others to first claim the vacancy. That's why she decided to use zk-SNARKs.
```

Comment:
Dla spójności myślę, że czas przeszły: [...] Alice was one of his friends who knew the solution.